conf/application.conf

```diff
@@ -1,5 +1,5 @@
 play.application.loader = "wiring.AppLoader"
-
+play.http.secret.key="a-very-secret-dev-key"
```

Comment:
As we just chatted about:
Yes but only to simplify the set-up for anyone new as that property is necessary to start play. It's not really related to that PR but I figured it shouldn't hurt